manifests/0000_03_cloud-credential-operator_00_namespace.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: v1
 kind: Namespace
```

Comment:
I don't understand what's happening with 9491d6070b4756a90's shift to runlevel 3.  Can you explain that, ideally in the commit message where future developers will be able to find it?